ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/diagnostic/IncompatibleResourceDiagnostic.java

```diff
@@ -1,72 +0,0 @@
-/*
- * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * WSO2 Inc. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package io.ballerina.openapi.service.mapper.diagnostic;
-
-import io.ballerina.tools.diagnostics.DiagnosticSeverity;
-import io.ballerina.tools.diagnostics.Location;
-
-import java.util.Optional;
-
-/**
- * This {@link IncompatibleResourceDiagnostic} represents the diagnostic that OAS not compatible with ballerina
- * implementation.
- *
- * @since 1.0.0
- */
-public class IncompatibleResourceDiagnostic implements OpenAPIMapperDiagnostic {
```

Comment:
I removed this since this seems to be redundant with `ExceptionDignostic`. If this suppose to represent specific type of diagnostic then I will revisit this when refactoring the diagnostics